packages/react-admin-core/src/RouterTabs.tsx

```diff
@@ -57,6 +57,16 @@ class RouterTabs extends React.Component<IProps> {
             return child.props.path;
         });
 
+        const rearrangedChildren = React.Children.toArray(this.props.children);
+        const defaultPathIndex = paths.indexOf("");
+
+        if (defaultPathIndex >= 0) {
+            const defaultPathChild = rearrangedChildren[defaultPathIndex];
+
+            rearrangedChildren.splice(defaultPathIndex, 1);
```

Comment:
the storybook-example works correctly even without rearranging the children,
can we have a story where re-arranging the children matters ?

Reply:
Navigation to /page2 (Tab "Page 2") doesn't work without rearranging the children.